src/main/java/com/istb/app/services/profesor/ProfesorServiceImpl.java

```diff
@@ -69,19 +79,19 @@ public Profesor findByCedula(String cedula) {
 	@Override
 	public Map<String, String> update(Profesor profesor, int id) {
 		Map<String, String> errorAttributes = new HashMap<>();
-		
+
 		Profesor _profesor = profesorRepository.findById(id).orElse(null);
-		
+
 		Profesor existeCorreo = profesorRepository.findByCorreoAndIdIsNot(profesor.getCorreo(), id);
-		
+
 		if (existeCorreo != null) {
```

Comment:
Con el cambio descrito en el comentario anterior, el `if` queda mas sencillo y sin `null`
```suggestion
		if (existeCorreo) {
```